code/modules/tgui/modules/ntos-only/configurator.dm

```diff
@@ -2,6 +2,7 @@
 	name = "NTOS Computer Configuration Tool"
 	ntos = TRUE
 	tgui_id = "Configuration"
+	available_to_ai = FALSE
```

Comment:
stealth nerf???

Reply:
This is mentioned in the PR body and the changelog that got deleted.

Reply:
This is a hardware configuration in the subsystems tab. Which would be fine if the A.I actually had any hardware. It doesn't. 

I don't believe this program is accessible in the general programs list either?

Reply:
ah, that'd make sense, yeah